scripts/e2e-test.sh

```diff
@@ -148,6 +149,13 @@ for dir in $TESTS; do
   fi
 done
 
+if [ "$GENERATE_LOGS" -ne 0 ]; then
```

Comment:
So we might get two lots of cluster dumps, if we're unlucky.

Reply:
I will fix, it is a bit messy at the moment.

Reply:
Fixed now